WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -77,4 +76,21 @@ extension XCTestCase {
         static let category = "iOS Test"
         static let tag = "tag \(Date().toString())"
     }
+
+    public func removeApp() {
+        let app = XCUIApplication()
```

Comment:
Nitpick. I _personally_ like using the convention of injecting the `XCUIApplication` instance instead of creating one in each method.

```swift
public func removeApp(_ app: XCUIApplication()) {
```

I never looked into whether it makes any difference at runtime, that is whether it takes noticeable extra time to the test runner to init `XCUIApplication`.

From a design point of view, though, I think it's good practice to do [as much Dependency Injection as possible](https://mokacoding.com/blog/explicit-dependencies/).

One further argument for the value of allowing injecting `app`. In the past, I worked on some UI tests that launched the messages app to test deep links. In that case, `XCUIApplication()` was _not_ the correct application to use to start and quit, we needed `XCUIApplication(<messages.app.bundle.id>)`.